documentation/docs/developer_guide/coding_standards.md

```diff
@@ -1 +1,160 @@
-# Coding standards
\ No newline at end of file
+## Foreword
```

Comment:
IDK - there is a one on the page there now - I suppose if this is to replace that entire page OR it might just be included like a header.txt might be in a HTML file